locallibrary/catalog/urls.py

```diff
@@ -0,0 +1,4 @@
+from django.urls import path
+from . import views
+urlpatterns = [
+]
```

Comment:
khai báo empty list như sau e nhé
```suggestion
urlpatterns = []

```